docs/topics/strings.md

```diff
@@ -132,4 +141,36 @@ use the following syntax:
 val price = """
 ${'$'}_9.99
 """
-```
\ No newline at end of file
+```
+## String formatting
+To deal with complex formatting requirements, use the `String.format()` function.  It uses a format string that defines 
+how the given arguments look like. The format string contains placeholders (`%`)  and
+[format specifiers](https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html). In the output, 
```

Comment:
As I recall we have agreed to have a link here only to `String.format` kdocs, and provide the link to Formatter from there.

Reply:
You're right. I corrected it.